### ISSUE TITLE
[Accessibility]: Screen Reader does not read the order for the list.

### ISSUE THREAD

**Issue by asthanarht** [tinymce/tinymce#5203]
Steps to reproduce.
1 . Start screen reader.
2. Navigate to the Tiny MCE tool bar strip with para graph option
3. press tab to enter in dropbox of paragraph like Heading 1, heading 2..etc
4. Listen narrator while navigate to paragraph dropbox value

Actual Result:
Screen Reader is not reading order  list  in the dropdown list of paragraph ,table , 11pt button.
Similar issue observed with Chrome + NVDA.
Expected Result:

Screen Reader should read in an order list of the dropdown list of paragraph ,table , 11pt button..

![image](https://user-images.githubusercontent.com/4268434/66688479-acedc500-ec3b-11e9-8527-c5f5deb9d1c9.png)

User Impact:

If the screen reader is not reading the list in an ordering way user will not able to know on which number the content belongs to.

**Comment by lnewson** [comment]
@asthanarht it appears as though you are using TinyMCE 4 which does have a few known accessibility issues. However, TinyMCE 4 is only receiving critical issues now, so can you I ask you to see if you can reproduce this issue on TinyMCE 5?

I should note TinyMCE 5 had a much larger focus on accessibility during its development. So while I only have access to VoiceOver at the moment, I believe this issue has been resolved in TinyMCE 5.

**Comment by yacodes** [comment]
We are closing this as there have been no additional details provided in the last month. If you can provide the details requested above, then we'd be happy to re-open this issue and investigate.
